src/util/string.h

```diff
@@ -357,6 +357,16 @@ inline std::string to_string(T val)
 	return oss.str();
 }
 
+/// Returns A string representing the value \p val.
+template <typename T>
+inline T from_string(const std::string &str)
+{
+	std::stringstream tmp(str);
+	T t;
+	tmp >> t;
+	return t;
+}
+
```

Comment:
This is placed between `to_string` and some `to_string` forwarders, breaking up two related sections.  Also, `stoi64` should forward to `from_string` just like `i64tos` forwards to `to_string` (or just replace all occurrences of `stoi64` with `from_string` if there are only a few of them).